src/Interpreters/InstrumentationManager.h

```diff
@@ -0,0 +1,154 @@
+#pragma once
+
+#include "config.h"
+
+#if USE_XRAY
+
+#include <vector>
```

Comment:
I don't think we have a preferred include order in the codebase, but what do you think about the following order?
```cpp
<CH_includes>

<external-libraries>

<cpp-libraries>
```

Reply:
Same, I don't think we ahve a preferred include order apart of alphabetically sorting the groups. I'll change it as per your suggestion